rpcserver.go

```diff
@@ -3964,6 +3965,25 @@ func (r *rpcServer) ListChannels(ctx context.Context,
 			continue
 		}
 
+		// First, parse the hex-encoded public key into a full in-memory
+		// public key object we can work with for querying.
+		pubKey, err := route.NewVertexFromStr(string(nodePubBytes))
+		if err != nil {
+			return nil, err
+		}
+
+		// With the public key decoded, attempt to fetch the node
+		// corresponding to this public key. If the node cannot be
+		// found, then an error will be returned.
+		node, err := graph.FetchLightningNode(pubKey)
+		switch {
+		case err == channeldb.ErrGraphNodeNotFound:
+			return nil, status.Error(codes.NotFound, err.Error())
+		case err != nil:
+			return nil, err
+		}
+		channel.Alias = node.Alias
+
```

Comment:
wondering if we should perhaps do something [like this](https://github.com/lightningnetwork/lnd/pull/7001#discussion_r991275083) here too so that users can opt out of the slightly slower call